cmd/print/print.go

```diff
@@ -77,7 +78,12 @@ func printOneDocument(printerName, documentName string, lines []string) error {
 	}
 	defer p.Close()
 
-	err = p.StartDocument(documentName, "RAW")
+	dataType, err := p.PrintDataType()
+	if err != nil {
+		return err
+	}
+
+	err = p.StartDocument(documentName, dataType)
```

Comment:
I suspect you had some problem with this code always passing "RAW" to StartDocument. What is the problem? How can I reproduce it? Can you create an issue about this problem and make this PR fix the problem?

Reply:
The problem is: i had a printer that doesnt accept RAW datatype, the problem is that print only accepts XPS documents.

Reply:
But what is the data that you need to write to the printer when you pass "XPS_PASS" to it? Can I send text data? Will it print properly? Does every printer that reports PRINTER_DRIVER_XPS will work with "XPS_PASS"? Are there any references to all of this somewhere on the Internet?

Maybe we could just introduce another flag for the program that describes data type, and default it to "RAW" instead?

Reply:
Normal string data, you can send normal text, its print properly, the driver do the work, and Yes every printer with PRINTER_DRIVER_XPS will work with XPS_PASS

And yes, we can do other flag for program to describe data type and set RAW as default

Reply:
Thank you for explaining. I will wait for you to address my other comments now.

Alex